src/funding/funding.ts

```diff
@@ -336,18 +336,18 @@ export class Funding {
     this.userFundingTokenAllowance = await contract.allowance(this.ethereumService.defaultAccountAddress, this.deal.daoDepositContracts.get(this.firstDao).address);
   }
 
-  @computedFrom("ethereumService.defaultAccountAddress", "deal.daoRepresentedByCurrentAccount", "deal.primaryDao")
+  @computedFrom("ethereumService.defaultAccountAddress", "deal.daoRepresentedByCurrentAccount")
   public get firstDao() : IDAO{
     return this.deal.isRepresentativeUser ? this.deal.daoRepresentedByCurrentAccount : this.deal.primaryDao;
   }
 
-  @computedFrom("ethereumService.defaultAccountAddress", "deal.daoRepresentedByCurrentAccount", "deal.primaryDao")
+  @computedFrom("ethereumService.defaultAccountAddress", "deal.daoRepresentedByCurrentAccount")
```

Comment:
Should be isRepresentativeUser be in computedFrom instead of defaultAccountAddress?